TUM Campus App/CollectionViewLayouts/ColumnsFlowLayout.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ColumnsFlowLayout.swift
+//  Campus
+//
+//  Created by Till on 30.06.18.
+//  Copyright © 2018 LS1 TUM. All rights reserved.
+//
+
+import UIKit
+
+class ColumnsFlowLayout: NSObject, UICollectionViewDelegateFlowLayout {
+    
+    let delegate: TUMInteractiveDataSource?
+    let margin: CGFloat = 20.0
+    
+    var isPhone: Bool {
+        return UIDevice.current.userInterfaceIdiom == .phone
+    }
+    
+    var isLandscape: Bool {
+        return UIDevice.current.orientation.isLandscape
+    }
+    
+    var columns: CGFloat {
```

Comment:
Using a CGFloat here does't seem right. Sure it makes further computations easier but...

Reply:
You’re right. Changed it.